File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    var answer = 0;
    
    let arr= [0, 1];
    
    for(let i=2; i<=n; i++) {
        const res = (arr[i-1] + arr[i-2]) %1234567;
        
        arr.push(res);
    }
    
    answer = arr[n];
    
    
    return answer;
}

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    
    vector<int> vec;
    
    vec.push_back(0);
    vec.push_back(1);
    
    for(int i=2; i<=n; i++) {
        const int res = (vec[i-1] + vec[i-2]) %1234567;
        
        vec.push_back(res);
    }
    
    answer = vec[n];
    
    return answer;
}